### PR TITLE
Improve getting started guide

### LINKED ISSUE #204: image-builder's Makefile does not have build-do-default
In section https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/blob/master/docs/getting-started.md#building-images it says to execute 

```
Run the Make target to generate DigitalOcean images.

$ make build-do-default
```

But the Makefile does not have such a rule.

Ref. https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/Makefile#L187

### PR DESCRIPTION
**Pull request by timoreimann**
**What this PR does / why we need it**:
Fixes several doc bugs in the getting started guide.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #204

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -27,19 +27,19 @@ doctl auth init --access-token ${DIGITALOCEAN_ACCESS_TOKEN}
...
-Change directory to images/capi within the image builder repository
+Change directory to images/capi within the image builder repository:
 
     $ cd image-builder/images/capi
 
-Run the Make target to generate DigitalOcean images.
+Choose a DigitalOcean image build target from the list returned by `make | grep build-do` and generate a DigitalOcean image (choosing Ubuntu in the example below):
 
-    $ make build-do-default
+    $ make build-build-do-ubuntu-2004
```
Comment (MorrisLaw) [inline docs/getting-started.md:40]: nice!

File: docs/getting-started.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f https://docs.projectc
 Deploy DigitalOcean Cloud Controller Manager
 
 ```bash
-kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f examples/digitalocean-ccm.yaml
+kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f examples/digitalocean-cloud-controller-manager.yaml
```
Comment (MorrisLaw) [inline docs/getting-started.md:96]: nice catch here too

**Review by MorrisLaw — APPROVED** [review]
Nice work. 

/lgtm

**Review by cpanato — APPROVED** [review]
/lgtm

I'm fine with the change but lets see with others think
/hold

**Pull request merged**